23homework.sql

```diff
@@ -0,0 +1,108 @@
+-- Tworzy tabelę pracownik(imie, nazwisko, wyplata, data urodzenia, stanowisko). W tabeli mogą być dodatkowe kolumny, które uznasz za niezbędne.
+CREATE TABLE pracownicy (
+	id_pracownika int primary key auto_increment not null,
```

Comment:
not null jest niejawnie dodane jak wstawiasz primary key, wiec mozna je tu usunac ;)